geopandas/io/tests/test_file.py

```diff
@@ -24,7 +24,7 @@
 try:
     import pyogrio
 
-    PYOGRIO_GE_07 = Version(pyogrio.__version__) > Version("0.6.0")
+    PYOGRIO_GE_07 = Version(pyogrio.__version__) >= Version("0.7.0")
```

Comment:
we've now got this in `io.tests.test_file` and `io.file`, does it make sense to consolidate in `_compat`?

Reply:
We're using a simpler version check for fiona and pyogrio in the tests, presumably because we control the testing environment, whereas the logic in `io.file` tries to do a better job of logging errors to the console when imports fail.

Since we need to also load `pyogrio` and `fiona` packages when executing the test file, it seemed cleaner to import them directly and then also set the version variables.

As for moving that logic to `compat`, I had assumed there was an intentional reason for first placing it into `io.file` instead, possibly planning ahead for a future where both IO engines are optional.

Reply:
Yes, we can't move this to compat as it would make it more complicated to load geopandas without an IO engine (which is possible now).